src/App.tsx

```diff
@@ -15,71 +16,82 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+interface State {
+  selectedGood : string;
+}
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  handleAddGood = (word: string) => {
+    this.setState({
+      selectedGood: word,
+    });
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  handleRemoveGood = () => this.setState({ selectedGood: '' });
```

Comment:
same here, your methods should perfectly describe what they do. It is important for the people that are going to work with the same piece of code - they don't want your variables lying to them 😄 
```suggestion
  unselectGood = () => this.setState({ selectedGood: '' });
```